src/client/components/EventsPage/EventsPage.component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { StudyGroupLogo } from '../StudyGroupLogo/StudyGroupLogo';
+import './EventsPage.styles.css';
+
+function EventsPage() {
+  return (
+    <div>
+      <div>Study Groups for Class15</div>
+      <div className="main-container">
+        <div className="top-part">
+          <div className="logo-part">
+            <div className="event-logo">
+              <StudyGroupLogo />
+            </div>
+            <div className="right-part">
+              <div className="event-group">Elastic - Week 46</div>
+              <div className="event-date">Wednesday | 11 Nov 20 | 10:00</div>
+              <div className="mentor-name">mentors name</div>
+              <div className="event-link">link to registoration</div>
+            </div>
```

Comment:
This whole `div.right-part` should be a list of events that come through props. You might have no events, 1 event, or more than one just as in Figma.